Formula/kafka.rb

```diff
@@ -6,6 +6,11 @@ class Kafka < Formula
   sha256 "3fa380ae5d1385111ee9c83b0d1806172924ffec2e29399fd1a42671a97492c6"
   license "Apache-2.0"
 
+  livecheck do
+    url "https://kafka.apache.org/downloads"
+    regex(/href=.*?kafka[._-]v?(\d+(?:\.\d+)+)-src\.t/i)
```

Comment:
```suggestion
    regex(/href=.*?kafka[._-]v?(\d+(?:\.\d+)+)-src\.t/i)
```

We use the Scala 2.13 tarball in the Formula (instead of the `-src` tarball), so the regex should match the tarballs with a filename that includes two versions separated by a hyphen (`-`). The Scala version will change over time, so the suggested regex above doesn't match a specific version (as it would quietly break and need to be manually updated eventually).

Reply:
make sense.

Reply:
Technically speaking, those scala flavors are cross build artifacts, so tracking one vs tracking another would be the same thing.

Also scala 3 is incompatible with scala 2.x (not sure how that would play out)